src/main/java/core/basesyntax/service/ValidatorImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service;
+
+public class ValidatorImpl implements Validator {
+    public static final int OPERATION = 0;
+    public static final int FRUIT_NAME = 1;
+    public static final int QUALITY = 2;
+    public static final int SIZE_TRANSLATION = 3;
+    public static final int ZERO = 0;
+
+    public boolean checkOperation(int money) {
+        if (money < ZERO) {
+            throw new RuntimeException("Incorrect input data");
+        }
+        return true;
+    }
+
+    public boolean checkInputData(String[] data) {
+        if (data.length != SIZE_TRANSLATION
+                || (data[FRUIT_NAME].isBlank()
+                || data[FRUIT_NAME].isEmpty())
+                || Integer.parseInt(data[QUALITY]) < 0
+                || !(data[OPERATION].equals("b")
+                || data[OPERATION].equals("s")
+                || data[OPERATION].equals("p")
+                || data[OPERATION].equals("r"))) {
```

Comment:
Maybe it would be better to move check for operation equality to b or s or p or r to single method.
If we will have more different operations(for example 10 operations) in future our method checkInputData would have a lot of simple rows.
Or we can use array or enum to store operations names and check if operation in data equals to some value from array.